tests/stats_extlog.rs

```diff
@@ -217,12 +222,42 @@ fn test_extloggable_setto() {
 fn basic_extloggable_grouped_by_string() {
     let (logger, mut data) = create_logger_buffer(SLOG_TEST_STATS);
 
-    xlog!(logger, ThirdExternalLog { name: "bar".to_string() });
-    xlog!(logger, ThirdExternalLog { name: "foo".to_string() });
-    xlog!(logger, ThirdExternalLog { name: "bar".to_string() });
-    xlog!(logger, ThirdExternalLog { name: "bar".to_string() });
-    xlog!(logger, ThirdExternalLog { name: "bar".to_string() });
-    xlog!(logger, ThirdExternalLog { name: "foo".to_string() });
+    xlog!(
```

Comment:
Not sure this is clearer. I assume it was forced by rustfmt? :-(

Reply:
Yes.  Personally I prefer the more compact form.